compiler/solidity_test.go

```diff
@@ -134,13 +134,13 @@ func TestDownloadSolidityCompiler(t *testing.T) {
 	}
 	defer os.RemoveAll(dst1)
 
-	if err := DownloadSolidity("0.5.5", dst1, true); err != nil {
+	if err := DownloadSolidity("0.8.19", dst1, true); err != nil {
```

Comment:
Maybe we can extract `0.8.19` to some global constant, and reuse it accros the codebase (not only in this test if needed). WDYT?